src/openrct2/object/TerrainEdgeObject.h

```diff
@@ -18,6 +18,7 @@ class TerrainEdgeObject final : public Object
     rct_string_id NameStringId{};
     uint32_t IconImageId{};
     uint32_t BaseImageId{};
+    uint32_t numImagesLoaded = 0;
```

Comment:
Could default initialize it as above.

Reply:
It should also be TitleCase.

Reply:
> Could default initialize it as above.

Initialising integers with `{}` is ugly, IMO.

Reply:
> Initialising integers with `{}` is ugly, IMO.

Maybe, but the other integers in the file are also default initialised. Consistency is good practice.

Reply:
Then we should change the code style, IMO. Anyway, I changed it.